tutorial/airfoilopt/multipoint/airfoil_multiopt.py

```diff
@@ -17,8 +17,8 @@
 # rst Imports (end)
 parser = argparse.ArgumentParser()
 parser.add_argument("--output", type=str, default="output")
-parser.add_argument("--opt", type=str, default="SLSQP", choices=["SLSQP", "SNOPT"])
-parser.add_argument("--gridFile", type=str, default="n0012.cgns")
+parser.add_argument("--opt", type=str, default="SNOPT", choices=["SLSQP", "SNOPT"])
```

Comment:
I think we always opted to leave `SLSQP` as default because it ships with pyOpstsparse and it is not proprietary like `SNOPT`

Reply:
Oh yeah good catch, didn't mean to push that, fixed now